`-Wsign-compare` warning in pdnsutil

- Program: Authoritative,
 - Issue type: Bug report

### Short description
```
pdnsutil.cc: In function ‘int main(int, char**)’:
pdnsutil.cc:2373:24: warning: comparison of integer expressions of different signedness: ‘int’ and ‘std::vector<std::__cxx11::basic_string<char> >::size_type’ {aka ‘long unsigned int’} [-Wsign-compare]
     for(int n=1; n + 2 <= cmds.size(); n+=2)
                  ~~~~~~^~~~~~~~~~~~~~
```

This is possible introduced with the merge of the LMDB backend.

--- a/pdns/pdnsutil.cc
+++ b/pdns/pdnsutil.cc
@@ -2370,7 +2370,7 @@ try
     if(cmds[1]==".")
       cmds[1].clear();
 
-    for(int n=1; n + 2 <= cmds.size(); n+=2)
+    for(size_t n=1; n + 2 <= cmds.size(); n+=2)
       loadZone(DNSName(cmds[n]), cmds[n+1]);
     return 0;
   }

